modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -300,12 +301,29 @@ protected ObjectDefinition randomObjectDefinition() throws Exception {
 						indexed = false;
 						indexedAsKeyword = false;
 						label = Collections.singletonMap("en_US", "Column");
+						localized =
+							objectDefinition.getEnableLocalization() &&
+							!objectDefinition.getSystem() &&
+							RandomTestUtil.randomBoolean();
```

Comment:
@seltong I changed my mind and we don't need to put this `RandomTestUtil.randomBoolean()`